plugin/slime.vim

```diff
@@ -153,8 +153,8 @@ function! s:VimterminalSend(config, text)
   " Ideally we ought to be able to use a single term_sendkeys call however as
   " of vim 8.0.1203 doing so can cause terminal display issues for longer
   " selections of text.
-  for l in split(a:text,"\n",1)
-    call term_sendkeys(bufnr,l."\<cr>")
+  for l in split(a:text,'\n\zs')
+    call term_sendkeys(bufnr,substitute(l,'\n',"\r",''))
```

Comment:
Replace the first `\n` with `\r`?

Reply:
Yes.  The flag could have 'g' since we want to replace all \n with \r, but the effect of the split above is that there is never more than 1 \n in the string so the 'g' would have no effect.